third-party/sysdeps/linux/amd-mesa/patch_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Patches installed binaries from the external build system.
+# Args: install_dir patchelf_binary
+set -e
+
+PREFIX="${1:?Expected install prefix argument}"
+PATCHELF="${PATCHELF:-patchelf}"
+THEROCK_SOURCE_DIR="${THEROCK_SOURCE_DIR:?THEROCK_SOURCE_DIR not defined}"
+Python3_EXECUTABLE="${Python3_EXECUTABLE:?Python3_EXECUTABLE not defined}"
+
+"$Python3_EXECUTABLE" "$THEROCK_SOURCE_DIR/build_tools/patch_linux_so.py" \
+  --patchelf "${PATCHELF}" --add-prefix rocm_sysdeps_ \
+  $PREFIX/lib/libgallium_drv_video.so \
+  $PREFIX/lib/libva.so \
```

Comment:
Getting error while installing amdrocm-sysdeps
- nothing provides librocm_sysdeps_va.so.2(AMDROCM_SYSDEPS_1.0)(64bit) needed by amdrocm-sysdeps7.1.0-7.1.0-crdnnh.x86_64 from @commandline

/usr/lib/rpm/elfdeps --provides librocm_sysdeps_va.so.2
libva.so.2(AMDROCM_SYSDEPS_1.0)(64bit)
libva.so.2(VA_API_0.32.0)(64bit)
libva.so.2(VA_API_0.33.0)(64bit)
libva.so.2(VA_API_0.33.0)(64bit)
librocm_sysdeps_va.so.2()(64bit)

@kiritigowda Rather than using patchelf to edit the libraries,  the libraries should be created with the updated names. The source code need to be edited to get the updated name
Pls refer the patch :https://github.com/ROCm/TheRock/pull/2456
cc @nunnikri @stellaraccident